spec/rspec/support/object_inspector_spec.rb

```diff
@@ -0,0 +1,74 @@
+require 'spec_helper'
```

Comment:
Our `.rspec` file has a flag to require this automatically so this line isn't needed.